app/sql.js

```diff
@@ -914,9 +914,13 @@ async function updateToLokiSchemaVersion1(currentVersion, instance) {
     profileAvatar: 'images/loki/loki_icon.png',
   };
 
-  await initConversation(publicChatData);
-  await initConversation(newsRssFeedData);
-  await initConversation(updatesRssFeedData);
+  const autoJoinLokiChats = false;
+
+  if (autoJoinLokiChats) {
+    await initConversation(publicChatData);
+    await initConversation(newsRssFeedData);
+    await initConversation(updatesRssFeedData);
```

Comment:
Do we really want to remove the RSS feeds too? My understanding, the messenger ALWAYS required and that's why it can't be closed.

Reply:
RSS feeds need to be removed since the fetching is not being done through proxy requests

Reply:
Oh, I probably can route them through the file server proxy if desired (if HF mentioned today is delayed)